src/Mesh/babylon.mesh.vertexData.ts

```diff
@@ -594,6 +594,7 @@
             var width = 1;
             var height = 1;
             var depth = 1;
+            var faceUV = options.faceUV || new Array(6);
```

Comment:
oki
I'll do both marks tomorrow when I'm back on my computer, unless you prefer to do it right now.